scripts/appConfig.ts

```diff
@@ -12,6 +12,14 @@ if (appConfig.shortTimeFormat == null) {
     appConfig.shortTimeFormat = 'HH:mm'; // 24h format
 }
 
+if (appConfig.view.dateformat == null) {
+    appConfig.view.dateformat = 'MM/DD'; // 24h format
+}
+
+if (appConfig.view.timeformat == null) {
+    appConfig.view.timeformat = 'hh:mm'; // 24h format
```

Comment:
Previously `HH:mm` was commented as being "24h format". Are you sure this is 24h format too?

Reply:
no is not... i just miss the comment